From 41d216ca7993991d32a077d0650ccfad8a846942 Mon Sep 17 00:00:00 2001
From: Julien Lepiller <julien@lepiller.eu>
Date: Sat, 25 Feb 2023 10:02:11 +0100
Subject: [PATCH] Remove annotation dependencies

This is used to prevent a failure when guix creates the Maven directory
structure and fails to find these dependencies.
---
 guava/pom.xml | 12 ------------
 1 file changed, 12 deletions(-)

diff --git a/guava/pom.xml b/guava/pom.xml
index 81a2005..5237047 100644
--- a/guava/pom.xml
+++ b/guava/pom.xml
@@ -31,18 +31,6 @@
       <groupId>com.google.code.findbugs</groupId>
       <artifactId>jsr305</artifactId>
     </dependency>
-    <dependency>
-      <groupId>org.checkerframework</groupId>
-      <artifactId>checker-qual</artifactId>
-    </dependency>
-    <dependency>
-      <groupId>com.google.errorprone</groupId>
-      <artifactId>error_prone_annotations</artifactId>
-    </dependency>
-    <dependency>
-      <groupId>com.google.j2objc</groupId>
-      <artifactId>j2objc-annotations</artifactId>
-    </dependency>
     <!-- TODO(cpovirk): does this comment belong on the <dependency> in <profiles>? -->
     <!-- TODO(cpovirk): want this only for dependency plugin but seems not to work there? Maven runs without failure, but the resulting Javadoc is missing the hoped-for inherited text -->
   </dependencies>
-- 
2.38.1

